crates/bevy_gizmos/src/config.rs

```diff
@@ -13,6 +13,17 @@ use std::{
     ops::{Deref, DerefMut},
 };
 
+/// An enum configuring how line joins will be drawn.
+#[derive(Debug, Copy, Clone, Reflect, PartialEq, Eq, Hash)]
+pub enum GizmoLineJoint {
+    /// Extends both lines at the joining point until they meet in a sharp point.
+    Miter,
+    /// Draws a round corner with the specified resolution between the two lines.
```

Comment:
What units are the resolution in?

Reply:
They count the triangles used to draw the arc. I'll add that information to the docs.